src/w3c/defaults.js

```diff
@@ -43,19 +25,21 @@ const w3cDefaults = {
       url: "https://www.w3.org/",
     },
   ],
-  addSectionLinks: true,
 };
 
 export function run(conf) {
   if (conf.specStatus === "unofficial") return;
   // assign the defaults
+  const lint = conf.lint && {
```

Comment:
Ah, sorry, we should support undefined `conf.lint` situation. So:

```suggestion
  const lint = conf.lint === false ? conf.lint : {
```